roles/unbind-keycloak-apb/tasks/main.yml

```diff
@@ -20,13 +18,14 @@
     var: keycloak_auth_response
     verbosity: 2
 
--
-  name: delete client {{ _apb_bind_creds.resource }} in realm {{ namespace }}
+- set_fact: KEYCLOAK_REALM={{ _apb_bind_creds.bearer_installation.realm}}
```

Comment:
How this `_apb_bind_creds.bearer_installation.realm` passed? Or where it is saved?

Reply:
The broker passes it back through as part of the unbind API

Reply:
The `_apb_bind_creds` is the secret created when you do a bind.